tsl/test/expected/dist_ddl.out

```diff
@@ -479,7 +477,7 @@ ERROR:  operation not supported on distributed hypertable
 ALTER TABLE disttable SET SCHEMA some_unexist_schema;
 ERROR:  schema "some_unexist_schema" does not exist
 ALTER TABLE disttable SET SCHEMA some_schema;
-ERROR:  operation not supported on distributed hypertable
+ERROR:  permission denied for schema some_schema
```

Comment:
I guess this error happens now because the CREATE SCHEMA above lacks AUTHORIZATION for ROLE_1? Is this intentional, e.g., to test lack of authorization? Would be good to add some comments to explain the intention of the test in that case.

Reply:
I've updated the test and moved this clause back